### PR TITLE
Initial raspberry component

### PR DESCRIPTION
**Pull request by AntonNeld**

### CODE REVIEW

File: raspberry/src/coreconnection.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+    def stop(self):
+        self._reconnect = False
+        if self._ws:
+            self._ws.close()
+
+    def _get_heartbeat_msg(self):
+        return json.dumps({
+            "messageType": "heartbeat",
+            "component": "audio",
```
Comment (mzlarsson) [inline raspberry/src/coreconnection.py]: Fix component name here

**Review by mzlarsson — CHANGES_REQUESTED** [review]
There are a lot of similarities between this and the previous components. Maybe it is not for this time, but for the future we should try to gather these common functionalities in one place and then reuse from there. That would make components much smaller, allowing them to focus more on their business logic.

**Comment by AntonNeld** [comment]
> There are a lot of similarities between this and the previous components. Maybe it is not for this time, but for the future we should try to gather these common functionalities in one place and then reuse from there. That would make components much smaller, allowing them to focus more on their business logic.

Makes sense, I agree

File: raspberry/src/coreconnection.py (hunk 1 above)
Comment (AntonNeld) [reply raspberry/src/coreconnection.py]: Done

**Comment by AntonNeld** [comment]
Made one more commit after addressing the comment, since it rounded out the PR nicely. But that was the last one (unless you have more comments)

File: ledController/src/commandhandler.py [hunk 2]
```diff
@@ -24,7 +24,7 @@ def _create_error_msg(self, msg):
         return result
     
     def initial_message(self):
-        return {"client": "raspberry"}
+        return {"client": "ledController"}
```
Comment (mzlarsson) [inline ledController/src/commandhandler.py:27]: If you change to snake case in the other PR, do so here as well. If not, ignore this comment.

**Review by mzlarsson — APPROVED** [review]

**Pull request merged**